web/src/components/predictions/utils.js

```diff
@@ -0,0 +1,129 @@
+/** @jsx jsx */
+import { jsx, Text } from "theme-ui";
+import firebase from "gatsby-plugin-firebase";
+
+const calculateScore = (isMC, prediction, answer, range) => {
```

Comment:
Is there a reason you can't do export up here like `export const` instead of at the bottom?